includes/derivatives.inc

```diff
@@ -94,8 +94,7 @@ function islandora_large_image_create_jp2_derivative(AbstractObject $object, $fo
       $derivative_file = islandora_large_image_imagemagick_convert(
         $uploaded_file,
         "temporary://{$base_name}_JP2.jp2",
-        islandora_large_image_get_args($lossless)
-      );
+        array_merge(islandora_large_image_get_args($lossless), array("-depth 8")));
```

Comment:
Should only really set the depth if it is less than 8...  If it were higher in the source, it would be impossible to produce a lossless image, were the flag passed.